Common/Brokerages/FTXBrokerageModel.cs

```diff
@@ -27,6 +28,14 @@ namespace QuantConnect.Brokerages
     /// </summary>
     public class FTXBrokerageModel : DefaultBrokerageModel
     {
+        private readonly OrderType[] _supportedOrderTypes =
```

Comment:
> OrderType[]

Let's replace this for a `HashSet<OrderType>` 👍

Reply:
Thank you for the review. I've made the requested change.